app/routes/reminders/reminder.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params) {
```

Comment:
I'd prefer to see a query to the store to get back a particular reminder - check out an example [here](https://guides.emberjs.com/v2.11.0/routing/specifying-a-routes-model/#toc_dynamic-models) ( particularly the part in the routes/photo code snippet)